R/tag_query.R

```diff
@@ -1161,6 +1169,20 @@ removeFromSet <- function(set, vals) {
   # removes the call to `unique()` with `setdiff`
   set[match(set, vals, 0L) == 0L]
 }
+isNonConformClassValue <- function(classVal) {
+  is.null(classVal) ||
+  anyNA(classVal) ||
+  is.list(classVal) ||
```

Comment:
Given the other checks I don't think this is necessary

```suggestion
  anyNA(classVal) ||
```